hal/armv7m/imxrt/117x/console.c

```diff
@@ -156,3 +177,20 @@ void _hal_consoleInit(void)
 	/* Enable TX and RX */
 	*(console_common.uart + uart_ctrl) |= (1 << 19) | (1 << 18);
 }
+#endif
+
+
+void _hal_consoleInit(void)
+{
+	unsigned int rtt_enable = 0u;
+
+#if RTT_ENABLED && !ISEMPTY(RTT_CONSOLE_KERNEL)
+	rtt_enable = 1u;
+#endif
+
+	_hal_rttInit(rtt_enable);
```

Comment:
Perhaps it would be cleaner to just move this define based check to _hal_rttInit instead? Then we do not need `enable` argument